pkgs/applications/networking/browsers/firefox/mobile-config.nix

```diff
@@ -9,111 +9,43 @@ let
   pkg = fetchFromGitLab {
     owner = "postmarketOS";
     repo = "mobile-config-firefox";
-    rev = "ff2f07873f4ebc6e220da0e9b9f04c69f451edda";
-    sha256 = "sha256-8wRz8corz00+0qROMiOmZAddM4tjfmE91bx0+P8JNx4=";
+    rev = "3fbea365e9ea7bcd317d5d4ef7723f994bbdc381";
```

Comment:
Any reason we're not updating to the latest version? https://gitlab.postmarketos.org/postmarketOS/mobile-config-firefox/-/tags/4.6.0

Reply:
I grabbed the last commit from master. But we can use that tag instead.

Reply:
Updated to 4.6.0.